.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+   extends: ['frost-standard'],
```

Comment:
This line should not have the array brackets but instead be:

extends: 'frost-standard',